### PR TITLE
EUI-3587: Fix "Print URL" pipe for rewriting remote Print Service URLs

### PR DESCRIPTION
**Pull request by Daniel-Lam**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-3587

### Change description ###
Pull request to create a PR environment for testing bugfix for EUI-3587, which fixes the pipe to rewrite remote Print Service URLs to the correct "local" version to be used for the front-end, irrespective of environment-based variations of the remote domain name.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/app/services/ccd-config/ccd-case.config.ts [hunk 1]
```diff
@@ -96,10 +96,6 @@ export class AppConfig extends AbstractAppConfig {
     return this.config.print_service_url;
   }
 
-  public getRemotePrintServiceUrl() {
```
Comment (paul-graham) [inline src/app/services/ccd-config/ccd-case.config.ts:99]: I think this makes it a breaking change...? That would mean it needs to go up a major version.

**Review by paul-graham — CHANGES_REQUESTED** [review]
I think the toolkit needs to have been a major release because it introduces a breaking change in the public API.

File: src/app/services/ccd-config/ccd-case.config.ts (hunk 1 above)
Comment (adnanakgun) [reply src/app/services/ccd-config/ccd-case.config.ts:99]: Agree with @paul-graham. Would this not break the build?

Comment (Daniel-Lam) [reply src/app/services/ccd-config/ccd-case.config.ts:99]: @paul-graham I've now re-introduced this in `app.config.ts` in the CCD UI Toolkit as a deprecated, no-op function that returns `undefined`, ahead of its removal in the next major release.

**Review by adnanakgun — APPROVED** [review]

**Review by paul-graham — APPROVED** [review]
All looks good. Definitely best to get ahead of a deprecation and ditch the redundant code. 😃

**Review by adnanakgun — APPROVED** [review]

**Review by paul-graham — APPROVED** [review]
Happy with this. 👍

**Pull request merged**